app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
If you are not going to modify nor extend this controllers, you can avoid creating them 😄